KwcNewsletter/Kwc/Newsletter/PluginInterface.php

```diff
@@ -7,4 +7,5 @@
     public function getNewsletterStatisticRows($newsletterRow, array $options = array());
     public function modifyRecipientsGridColumns(Kwf_Collection $columns, $gridType);
     public function modifyRecipientsSelect(Kwf_Model_Select $select, $gridType);
+    public function modifyReturnPath(Kwf_Mail $mail, Kwf_Component_Data $newsletter, Kwf_Model_Row_Interface $recipientRow);
```

Comment:
vorschlag: ändern wir das doch in modifyMail oder afterCreateMail oder so ähnlich

Reply:
:+1: